ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -783,13 +783,12 @@ <h3 class="title">
       <ng-container matColumnDef="spIndexOrder">
         <th class="table_header" mat-header-cell *matHeaderCellDef>Column Order</th>
         <td mat-cell *matCellDef="let element">
-          <div *ngIf="isIndexEditMode && element.get('spOrder').value">
+          <div *ngIf="isIndexEditMode && element.get('spColName').value">
             <input
               matInput
               class="order_input"
-              type="text"
+              type="number"
```

Comment:
Weird issue with PK edit , steps to reproduce -
1. Change the order of primary key and click on save
2. Add a column to PK and click on cancel.
You will see that the UI shows the reverted order.

Reply:
Solved